src/osha/oira/statistics/utils.py

```diff
@@ -84,6 +77,16 @@ def update_database(self, country=None):
         self.log_counts()
 
     def update_tool(self, country=None):
+        log.info("Table: tool")
+        latest_published_date = (
+            self.session_statistics.query(
+                sqlalchemy.func.max(SurveyStatistics.published_date)
+            ).first()[0]
```

Comment:
Theoretically `.first()` can return `None`.

That might not be a problem on a DB that already has data but I am wondering if this will hurt somebody playing with test data some day...

Reply:
In this case `.first()` returns a tuple (because of the `max` I think), so if there is no data it returns `(None,)` which works fine.